android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiUIWebView.java

```diff
@@ -1070,4 +1076,18 @@ protected void disableHWAcceleration()
 	{
 		Log.d(TAG, "Do not disable HW acceleration for WebView.", Log.DEBUG_MODE);
 	}
+
+	public void setSoftwareMode(Boolean value)
+	{
+		WebView webView = getWebView();
+		if (webView != null) {
+			softwareMode = value;
+			if (value) {
+				webView.setLayerType(View.LAYER_TYPE_SOFTWARE, null);
+			} else {
+				webView.setLayerType(View.LAYER_TYPE_HARDWARE, null);
```

Comment:
There are three layer types (LAYER_TYPE_HARDWARE, LAYER_TYPE_SOFTWARE and LAYER_TYPE_NONE), so a boolean may not be the best fit here

Reply:
I've change it to be `webview.layerType` and added constants for all three options. Will update the example in the first post in a bit